notebooks/02-quality-flags/TIKEBook_Lesson2.ipynb

```diff
@@ -0,0 +1,1011 @@
+{
```

Comment:
<p>As the reader I notice that this one looks different than the one above (with the red dots, straight from the .fits file). I believe it's because we said quality_bitmask="none" this time, right? We can be explicit about what we are doing and why - even if we just say it's to illustrate the point about quality flags.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>No, its because we are using lightkurve's plotting routine and not making the plot ourselves. the bitmask = none is to make sure we are not getting previously "cleaned" data. I'll add a markdown cell with a bit more info on why we are calling it this way. Can plot it manually we did before if we think that would be clearer.</p>

Reply:
<p><code>.scatter()</code>, in place of <code>.plot()</code> will also achieve this effect</p>